v3/src/components/slider/slider-model.ts

```diff
@@ -76,7 +76,9 @@ export const SliderModel = TileContentModel
         else return num
       }
 
-      if (self.multipleOf && self.increment) {
+      // The bounds check for date scale type happens in the use-slider-animation
+      // so we skip the bounds check if the axis is a date axis here
+      if (self.multipleOf && self.increment && self.scaleType !== "date") {
```

Comment:
```suggestion
      if (self.multipleOf && self.increment) {
```
As we discussed in sync review, once `setValidatedValue()` is available, this should no longer be necessary.